rclpy/rclpy/qos.py

```diff
@@ -381,20 +381,23 @@ class LivelinessPolicy(QoSPolicyEnum):
 # Alias with the old name, for retrocompatibility
 QoSLivelinessPolicy = LivelinessPolicy
 
+# module-level data members are not automatically documented by Sphinx. Use an
+# empty ``#:`` as a workaround. See
+# https://github.com/sphinx-doc/sphinx/issues/1063
```

Comment:
Rather than appending an empty `#:` as a workaround, I think it would be better to add some content to the doc comment.

It could be something minimal, for example,

```python
#: A good QoS profile for sensor data, using best effort reliability.
qos_profile_sensor_data = QoSProfile(**_rclpy.rclpy_get_rmw_qos_profile(
    'qos_profile_sensor_data'))
```

Note, it looks like Sphinx is happy to have the comment appearing the line before the variable.

Reply:
This is a good idea! I never used these profiles, but can try to get some info from rmw docs.